src/rgw/rgw_admin.cc

```diff
@@ -229,8 +229,8 @@ void _usage()
   cout << "   --reset-regions           reset regionmap when regionmap update\n";
   cout << "   --bypass-gc               when specified with bucket deletion, would trigger\n";
   cout << "                             object deletions by not involving GC\n";
-  cout << "   --keep-index-consistent   set this to keep the bucket index consistent, if the bucket\n";
-  cout << "                             deletion is triggered with bypass-gc set to true\n";
+  cout << "   --inconsistent-index      setting this would not bother to keep the bucket index consistent,\n";
```

Comment:
this text needs to be moved into the imperative or present tense "when specified with bucket deletion, ignore bucket index consistency (since it will be removed)"--can't be "would"